pkg/utils/namespaces.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+// deployNamespaceEnv is a constant for env variable DEPLOYMENT_NAMESPACE
+// which specifies the Namespace to watch.
```

Comment:
missed it in the ealier review: this isn't the namespace to watch (it's watching all namespaces)